# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

RRActionGoal action_goal
RRActionResult action_result
RRActionFeedback action_feedback
